Src/BootCamp.Chapter/PersonInfo.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class PersonInfo
+    {
+        public static void UserInfo()
+        {
+            //ask for first input of name
```

Comment:
Quite verbose, it's really clear what you did. Comments, espcially for beginner are useful, Well done :)